lightly/api/api_workflow_compute_worker.py

```diff
@@ -250,24 +250,8 @@ def create_compute_worker_config(
                 "creator": self._creator,
             }
         )
-        try:
-            response = self._compute_worker_api.create_docker_worker_config_vx(request)
-            return response.id
-        except ApiException as e:
-            if e.body is None:
-                raise e
-            eb = json.loads(e.body)
-            eb_code = eb.get("code")
-            eb_error = eb.get("error")
-            if str(e.status)[0] == "4" and eb_code is not None and eb_error is not None:
-                raise ValueError(
-                    f"Trying to schedule your job resulted in\n"
-                    f">> {eb_code}\n>> {json.dumps(eb_error, indent=4)}\n"
-                    f">> Please fix the issue mentioned above and see our docs "
-                    f"https://docs.lightly.ai/docs/all-configuration-options for more help."
-                ) from e
-            else:
-                raise e
```

Comment:
No need anymore to add a custom error message twice.

Reply:
We still do and get an even better link:
![image](https://github.com/lightly-ai/lightly/assets/20324507/4bba5218-4a9d-48ce-beee-dba3173d73cd)